server/schedule/metrics.go

```diff
@@ -50,19 +50,37 @@ var (
 			Buckets:   prometheus.ExponentialBuckets(0.01, 2, 16),
 		}, []string{"type"})
 
-	storeLimitGauge = prometheus.NewGaugeVec(
+	storeLimitAvailableGauge = prometheus.NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: "pd",
 			Subsystem: "schedule",
-			Name:      "store_limit",
-			Help:      "Limit of store.",
-		}, []string{"store", "type", "limit_type"})
```

Comment:
The metrics will not be compatible with the old version of grafana template, why can't it keep the original way?

Reply:
> The metrics will not be compatible with the old version of grafana template, why can't it keep the original way?

The old way has two `type`, `"take"` and `"available"`. I use  a new metrics `storeLimitAvailableGauge` to record the `"available"` type `storeLimitGauge`, but use a different metrics `storeLimitCostCounter` (it is a counter, not gauge) to record the original `"available"` type `storeLimitGauge` metrics.